CorsaSchoolManagementSystem/Views/dashboard.php

```diff
@@ -180,7 +180,7 @@
                             <?php foreach($studentDetails as $details): ?>
                             <tr>
                                 <td><img src="<?php echo BASE_URL . "/" .$details->picture?>" width="50" height="50" class="img-thumbnail" alt="<?php echo $details->firstname ."'s profile picture" ?>"/></td>
-                                <td><?php echo $details->firstname . ' ' . $details->middlename. ' ' .$details->lastname ?></td>
+                                <td><a href="<?php echo BASE_URL ?>/Views/studentDetails.php" class="text-decoration-none text-dark"><?php echo $details->firstname . ' ' . $details->middlename. ' ' .$details->lastname ?></a></td>
```

Comment:
You don't use views for application URLs, use controllers for this

Reply:
That was Sammy’s branch but after his commit to the branch he couldn’t find the pull request on his machine but I could find it on mine so I did the pr for him

Reply:
Okay! You can make the changes for him or just inform to do those changes

Reply:
Alright I wiil do that